dashboard/consts.py

```diff
@@ -79,3 +79,7 @@
     "dashboard",
     "gcs",
 }
+# 20MB
```

Comment:
nit: i wonder if we should put all the grpc related consts in a single file. I know there are some other configs that we override? like the keep alive timeouts.